packages/react-components/react-tags/src/components/Tag/renderTag.tsx

```diff
@@ -9,5 +9,17 @@ export const renderTag_unstable = (state: TagState) => {
   const { slots, slotProps } = getSlots<TagSlots>(state);
 
   // TODO Add additional slots in the appropriate place
-  return <slots.root {...slotProps.root} />;
+  return (
+    <slots.root {...slotProps.root}>
+      {slots.content && (
+        <slots.content {...slotProps.content}>
+          {slots.avatar && <slots.avatar {...slotProps.avatar} />}
+          {slots.icon && <slots.icon {...slotProps.icon} />}
+          {slots.primaryText && <slots.primaryText {...slotProps.primaryText} />}
+          {slots.secondaryText && <slots.secondaryText {...slotProps.secondaryText} />}
+        </slots.content>
+      )}
+      {slots.dismissButton && state.dismissable && <slots.dismissButton {...slotProps.dismissButton} />}
```

Comment:
Is there a reason why not make everything a single grid? or is it to support rtl for the dismiss button? you could potentially check if it's rtl with fluent context and get rid of the extra layer.

Reply:
With tag, maybe. I was consdering being able to remove the 'content' slot so that everything was a child of the root. But with the TagButton, the root needs 2 children (2 buttons), and the primary button needs to handle the layout (until subgrid :D)